app/build.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
     androidTestImplementation 'androidx.test:runner:1.1.0'
     androidTestImplementation 'androidx.test:rules:1.1.0'
+    implementation 'com.android.support:support-v4:28.0.0'
```

Comment:
What is this for?

Reply:
I believe it is for AlertDialog which shows a warning when you try to block someone